src/main/java/org/usfirst/frc4904/robot/commands/LowerHook.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+
+public class LowerHook extends MotorConstant {
+  public static final double DEFAULT_DOWN_SPEED = -0; // TODO: This is an untested value
+  public Motor tubeMotor;
+  public double speed;
+
+  /**
+   * Lower the deploy tube to hook onto the generator switch
+   * 
```

Comment:
We should always be using the same `tubeMotor` from `RobotMap`, so we'll only need one constructor that just calls `super("LowerHook", RobotMap.Component.tubeMotor, DEFAULT_DOWN_SPEED);`